### ISSUE TITLE
Track routing/pattern-learning accuracy over time (SONA/ReasoningBank)

### ISSUE THREAD

**Issue by nokhodian** [monoes/monomind#212]
## Context

The \`hooks intelligence\` / SONA+ReasoningBank system does real nearest-neighbor
pattern matching (cosine similarity over embeddings) with confidence scores that
adapt based on recorded trajectory outcomes (success/failure/partial). \`EWC++\`
consolidation is explicitly self-documented (see \`ewc-consolidation.ts\`'s header)
as a heuristic *inspired by* Elastic Weight Consolidation, not a real
implementation — no model, no gradients, no training loop.

\`monomind doctor\`'s "Routing Learning" check surfaces the actual measured
signal for whether this is working in practice:

\`\`\`
✓ Routing Learning: routing accuracy (last 100): no outcome data yet | adherence n/a (n=0)
\`\`\`

As of this checkout, there is **zero recorded outcome data** — the mechanism
is real and running, but nothing yet demonstrates it's actually improving
routing decisions here.

## Purpose of this issue

A standing, never-closed tracker to periodically capture the actual measured
routing accuracy / adherence numbers as they accumulate, so we build a real
longitudinal picture instead of relying on the mechanism description alone.

**How to update:** run \`monomind doctor\` (or \`monomind hooks route-outcomes\`
if a dedicated inspector exists by the time you read this) and post the
"Routing Learning" line — plus sample size \`n\` — as a new comment below.
Do not close this issue; it exists to accumulate history over time.

## Baseline (2026-08-21)

\`\`\`
✓ Routing Learning: routing accuracy (last 100): no outcome data yet | adherence n/a (n=0)
\`\`\`

n=0 — no outcome data recorded yet in this checkout.

**Comment by nokhodian** [comment]
Update (2026-08-22):

\`\`\`
✓ Routing Learning: routing accuracy (last 100): 100% [native n/a / js 100%] | adherence 0% (n=1)
\`\`\`

n=1 — first outcome recorded since the n=0 baseline (2026-08-21). Too small a sample to draw any conclusion yet (100%/0% at n=1 just means the one recorded outcome was correct on accuracy but didn't match the predicted route on adherence) — noting it here per this issue's own instructions so the history accumulates. Also observed this checkout is running CLI v2.9.9 pinned locally against a v2.9.22 install available (`monomind doctor`'s version-freshness warning) — worth keeping in mind if n stays low for a while, in case the outcome-recording path itself changed between those versions.

**Comment by nokhodian** [comment]
Update (2026-08-28):

```
Routing Learning: routing accuracy (last 100): 100% [native n/a / js 100%] trend +0% | adherence 0% (n=2)
```

n=2. This is one additional recorded outcome since the prior n=1 update; the sample remains far too small to infer a routing-quality trend. Fresh `monomind doctor` also reports the graph is fresh (0 commits behind).
